### ISSUE TITLE
What is Amazon ECS/Fargate used for?

### ISSUE THREAD

**Issue by sjockers** [technologiestiftung/giessdenkiez-de-dwd-harvester#92]
There's a provisioning script for ECS in this repo and a reference to Fargate in the readme. Is this actually meant to be used or is it legacy code/infrastructure? My understanding was that the resulting Docker image is used in this GH workflow. Or am I missing something?

**Comment by ff6347** [comment]
Yes the fargate/ecs stuff is legacy code and can be removed. We had it first running as aws fargate but moved it to GitHub Actions for more transparency. 

>Also, maybe related, I was wondering why the cron schedule is commented out for the harvesters – e.g. here: technologiestiftung/giessdenkiez-de@master/.github/workflows/rain.yml#L7-L8

The cron schedule is commented out because we use repository_dispatch actions. If a repo development is stale (no commits happening) a schedule action is paused after 60 (or was it 90) days. We moved to cronjobs running on pipedream.com using repo dispatch to be sure that all jobs keep running.
